src/circuit/gadget/ecc/chip/add.rs

```diff
@@ -190,14 +189,12 @@ pub(super) fn assign_region<C: CurveAffine>(
             .map(|(((x_p, x_q), y_p), y_q)| -> Result<(), Error> {
                 if x_q == x_p {
                     // x_q = x_p ⟹ A
-                    region.assign_advice(|| "set A", a, offset, || Ok(C::Base::one()))?;
+                    region.assign_advice(|| "set A", a, offset, || Ok(F::one()))?;
 
                     // Doubling case, λ = (y_q − y_p) / (x_q − x_p)
```

Comment:
This comment is not correct; give the equation for the tangent slope.

Reply:
Changed to `λ = 3(x_p)^2 / (2 * y_p)`